### PR TITLE
feat: distribution centres in a a map

### PR DESCRIPTION
**Pull request by machi1990**
##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added
Screenshot 2020-05-20 at 13 02 01

We have that little nice view, opening as draft as I need to bring in in the detail view so that when you click on a distribution marker it'll land you to the distribution centre detail edit/ view.

### CODE REVIEW

File: platform/client-admin/src/pages/DistributionCentrePage.tsx [hunk 1]
```diff
@@ -1,7 +1,75 @@
...
+    };
+
+    let distributionMarkers = distributionCentres
+      .map((distributionCentre, index) => {
+        const lat = parseInt(distributionCentre?.lat!);
+        const lng = parseInt(distributionCentre?.long!);
+        
+        totalLatLong.lat += lat;
+        totalLatLong.lng += lng;
+
```
Comment (machi1990) [inline platform/client-admin/src/pages/DistributionCentrePage.tsx:39]: Poor man's algo to find centre: let sum it all then divide by size.

**Comment by wtrocki** [comment]
@machi1990 It will be cool if we focus on area of max 60km and push the sample data. Paris, New York you name it.

**Comment by machi1990** [comment]
> @machi1990 It will be cool if we focus on area of max 60km and push the sample data. Paris, New York you name it.

To be honest I have not focused on sample data yet - the ones you see on map are stuffs I added manually. 

I think it will be easier to populate them once we have capability to create a distribution centre. Which is of course my next target.

**Comment by machi1990** [comment]
Added a distribution centre creation view. We can now play with admin and create real data.

**Pull request merged**